microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/UndeployDeployTest.java

```diff
@@ -118,43 +118,14 @@ public void testFaultToleranceSubsystemNotActivatedByNonMPFTDeployment(
     }
 
     /**
-     * @tpTestDetails Deploy MP FT application which initializes MP FT/Hystrix, undeploy and deploy different MP FT
-     *                application which changes Hystrix configuration.
-     * @tpPassCrit Hystrix configuration was changed.
+     * @tpTestDetails Deploy first MP FT application which initializes MP FT then deploy second MP FT
+     *                application which has different MP FT configuration.
+     * @tpPassCrit MP FT configuration is different for both deployments.
      * @tpSince EAP 7.4.0.CD19
      */
     @Test
     @InSequence(10)
-    public void testUndeployDeployChangesHystrixConfiguration(
```

Comment:
This type of test is still valid as we want to check that new deployment has different configuration.

Reply:
Well, there are similar tests here for example 'testSecondDeploymentDoesChangeConfiguration' which checks that deployments have different configurations even when deployed simultaneously. In the deleted test, deployments are deployed separately one after the other so I assumed that this test is 'weaker' and that it's covered by before mentioned test. WDYT?

Reply:
I think that we should still keep this one as it's different scenario. It should check that once MP FT deployment is undeployed, MP FT is deactivated.

Reply:
Ok, I am keeping the test.